### ISSUE TITLE
Jsonpath support for kops

### ISSUE THREAD

**Issue by chrislovecnm** [kubernetes/kops#1240]
As @yissachar mentioned in an issue.  It would be AWESOME to have jsonpath support in `kops`.  The requirements are exactly the same as how jsonpath support works in `kubectl`.

**Comment by kris-nova** [comment]
Please `-oyaml` too 😍

**Comment by kris-nova** [comment]
Also pointing to https://github.com/kubernetes/kops/issues/1077 - we would want encoding support there as well

**Comment by chrislovecnm** [comment]
Agreed, it would be good to mimic `kubectl` with json, table, and yaml support.  JSON is only handy to me when I am parsing it.

**Comment by stealthybox** [comment]
@adamcaron is interested in working on this

**Comment by chrislovecnm** [comment]
@adamcaron you want to start with just JSON support for the get commands and validate?

I will assign this to myself, as I cannot assign it to you.  This is probably multiple PRs:

1. json
2. yaml everywhere
3. jsonpath
4. go template
